cve_bin_tool/error_handler.py

```diff
@@ -243,6 +249,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
     UnknownArchiveType: 31,
     UnknownConfigType: 32,
     CVEDataMissing: 33,
+    InvalidSpdxError: 33,
```

Comment:
```suggestion
    InvalidSpdxError: 33,
```

Probably just give this a new number for disambiguation.  I don't really have these organized in any way so you can dump it at the bottom.